docs/spec/data-plane.md

```diff
@@ -91,6 +91,14 @@ delivered.
 CloudEvents received by Sink MAY have
 [Distributed Tracing Extension Attribute](https://github.com/cloudevents/spec/blob/v1.0/extensions/distributed-tracing.md).
 
+### Event reply header
+
+An event sender, including Source and Broker and Channel, SHOULD include
+the event reply header `K-Eventing-Event-Reply: response` in an event delivery
+request if it supports response events. If no event reply header is present
+or the header is present but does not include the value `response`,
```

Comment:
the value `response`  is strange.  What are the other values?

Reply:
Does it make more sense if the header name is `K-Eventing-Accept-Reply`?

Reply:
For now an empty value means not supported. I think string is safer than boolean for future extension just in case. WDYT?

Reply:
I agree 'response' is very strange for the value. I also would prefer a string as opposed to a bool.